minicluster/src/main/java/org/apache/accumulo/miniclusterImpl/MiniAccumuloClusterControl.java

```diff
@@ -111,6 +127,53 @@ public void adminStopAll() throws IOException {
     }
   }
 
+  private static TExternalCompactionList getRunningCompactions(ClientContext context)
+      throws TException {
+    Optional<HostAndPort> coordinatorHost =
+        ExternalCompactionUtil.findCompactionCoordinator(context);
+    if (coordinatorHost.isEmpty()) {
+      throw new TTransportException("Unable to get CompactionCoordinator address from ZooKeeper");
+    }
+    CompactionCoordinatorService.Client client = ThriftUtil.getClient(
+        new CompactionCoordinatorService.Client.Factory(), coordinatorHost.get(), context);
+    try {
+      TExternalCompactionList running =
+          client.getRunningCompactions(TraceUtil.traceInfo(), context.rpcCreds());
+      return running;
+    } finally {
+      ThriftUtil.returnClient(client, context);
+    }
+  }
+
+  public synchronized void startCoordinator(Class<? extends CompactionCoordinator> coordinator)
+      throws IOException {
+    if (coordinatorProcess == null) {
+      coordinatorProcess = cluster
+          ._exec(coordinator, ServerType.COMPACTION_COORDINATOR, new HashMap<>()).getProcess();
+      UtilWaitThread.sleep(1000);
+      // Wait for coordinator to start
+      TExternalCompactionList metrics = null;
+      while (null == metrics) {
```

Comment:
Awhile back, somebody (I think @jmark99) went through and standardized all our code on `x == null` instead of `null == x`. It'd be good to try to preserve that consistency going forward. We don't currently have an automated way to enforce this, but I think you might be one of the few who use this pattern.

```suggestion
      while (metrics == null) {
```

Reply:
I updated this in [851c2c3]